src/internal/utilities.ts

```diff
@@ -2,7 +2,7 @@ import * as types from '../types';
 import { Dictionary } from './formatting';
 
 /** Raf for node + browser */
-export const raf = typeof requestAnimationFrame === 'undefined' ? setTimeout : requestAnimationFrame.bind(window);
+export const raf = typeof requestAnimationFrame === 'undefined' ? function(cb: Function) {setTimeout(cb, 0)} : requestAnimationFrame.bind(window);
```

Comment:
Using `() => void` instead of `Function` is probably going to give us safer code.

Reply:
Your advice applied in the next commit.
Anyway, seems like variable `raf` lost type information ;-)